src/richie/apps/demo/defaults.py

```diff
@@ -6,13 +6,13 @@
 NB_OBJECTS = {
     "courses": 30,
     "course_organizations": 3,
-    "course_persons": 3,
+    "course_persons": 6,
     "course_subjects": 2,
     "person_subjects": 3,
     "organizations": 5,
     "licences": 5,
-    "persons": 10,
-    "blogposts": 6,
+    "persons": 100,
```

Comment:
It's a little bit too much for a demo, 50 seems more reasonable and just as relevant than 100. Demo site routine was already a little long to proceed before these changes, this is a demo site, not a stress test demo :)

Reply:
ok